cdap-docs/_common/common-build.sh

```diff
@@ -288,11 +288,11 @@ function copy_license_pdfs() {
 function set_mvn_environment() {
   check_build_rst
   cd ${PROJECT_PATH}
-  if [[ "${OSTYPE}" == "darwin"* ]]; then
+  if [[ ${OSTYPE} == "darwin"* ]]; then
     # TODO: hard-coded Java version 1.7
     export JAVA_HOME=$(/usr/libexec/java_home -v 1.7)
+    java -version
```

Comment:
This will still always return whichever Java version is returned first in your PATH, not necessarily the version in `JAVA_HOME`... if you use `${JAVA}` from `functions.sh`'s `cdap_set_java`, then it'll always be the version in `JAVA_HOME`. Since I don't handle the Mac case, you'll want to still do that and set JAVA_HOME on Mac before running the `cdap_set_java` function, followed by using `${JAVA} -version` here.